src/Resources/skeleton/crud/controller/Controller.tpl.php

```diff
@@ -0,0 +1,104 @@
+<?= "<?php\n"; ?>
+
+namespace <?= $namespace; ?>;
+
+use <?= $entity_full_class_name; ?>;
+use <?= $form_full_class_name; ?>;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("<?= $route_path; ?>", name="<?= $route_name; ?>_")
+ */
+class <?= $class_name ?> extends Controller
+{
+    /**
+     * @Route("/", name="index")
+     *
+     * @return Response
+     */
+    public function index()
```

Comment:
I have another trick one for you 😇. IF there is a custom repository class, I would rather type-hint it as an argument to the method:

```php
public function index(SourFoodRepository $repository)
{

}
```